### DISCUSSION TITLE
Sample stuff for testing

### DISCUSSION THREAD

**Discussion by ghost** [Peter-van-Tol/LiteX-CNC#37] [Show and tell]
Hi Peter,

I've started a repo here https://github.com/ozzyrob/litex-configs with some basic hal & pyvcp files for testing, without having to fully start linuxcnc. Includes the firmware I've generated for Linuxcnc 2.9

**Comment by Peter-van-Tol** [comment]
Thanks, will leave this issue open, so people can find your examples and tests.

**Comment by ghost** [comment]
No rpobs

Maybe a Discussion would be a good thing.
https://docs.github.com/en/discussions/quickstart
